ql/pom.xml

```diff
@@ -1109,6 +1105,7 @@
                   <include>com.esri.geometry:esri-geometry-api</include>
                   <include>org.roaringbitmap:RoaringBitmap</include>
                   <include>com.github.luben:zstd-jni</include>
+                  <include>org.locationtech.jts:jts-core</include>
```

Comment:
Since we are now bundling jts-core in the shaded jar we may also have to add relocation (see L1129) to avoid version conflicts in case other deps bring in the same classes.

Reply:
Good ssuggestion! Fixed it!